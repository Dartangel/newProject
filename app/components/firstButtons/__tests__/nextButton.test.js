import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import { FirstButtons } from '../index';

describe('FirstButtons', () => {
    test('FirstButtons snapshot', () => {
        const renderer = new ShallowRenderer();
        const snapshot = renderer.render(<FirstButtons />);
        expect(snapshot).toMatchSnapshot();
    });
});