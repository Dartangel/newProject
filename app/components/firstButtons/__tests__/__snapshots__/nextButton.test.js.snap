// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FirstButtons FirstButtons snapshot 1`] = `
<TouchableOpacity
  activeOpacity={0.2}
  onPress={[Function]}
  style={
    Object {
      "marginVertical": 40,
    }
  }
>
  <Text
    style={
      Object {
        "color": "#08DBA5",
        "fontSize": 35,
      }
    }
  />
</TouchableOpacity>
`;
